src/components/plot/plot-control-grid.tsx

```diff
@@ -38,7 +41,10 @@ const PlotControlGrid = ({
   const [gridProps, setGridProps] = useState({});
   const [userControlsExists, setUserControlExists] = useState<boolean>(false);
   const [userControlsReady, setUserControlReady] = useState<boolean>(false);
```

Comment:
I noticed an inconsistency in this state variable. we have `userControlsReady` and `setUserControlReady` (the `set` is not plural).

```
setUserControlReady --> setUserControlsReady
```

Can you change this to be consistent and update the places in this file where the set function is called?